docker/multi-process/scripts/init

```diff
@@ -5,6 +5,15 @@ export LC_ALL=en_US.UTF-8
 
 cd /app
 
+# Do we have a Rails secret yet?
+# If not, generate one, but persist it for the container's lifetime.
+if grep -q ^APP_SECRET_TOKEN=REPLACE_ME_NOW .env.example; then
```

Comment:
Ah gotcha. Yeah, that's trivial to add an env var check too. Will do in the next re-push.